### DISCUSSION TITLE
How to fix webview version?

### DISCUSSION THREAD

**Discussion by arasrezaei** [ionic-team/capacitor#3181] [Q&A] [answered]
is there any plugin that we could use to fix webview android for capacitor? like legacy crosswalk

**Comment by imhoffd** [comment, marked as answer]
No. We likely will never support this use case in Capacitor.

**Reply by arasrezaei** [reply]
What about those phones with outdated webview ?
Or those without proper system webview like in custom rooms?

**Reply by dotNetkow** [reply]
You could build with Cordova if you need to support those. In order to innovate and keep the project moving forward, the team can only support ios/android versions (and thus, WebViews) back so far.  [From the docs](https://capacitorjs.com/docs/getting-started/dependencies):

As a rule, the latest version of Capacitor always supports at least the last two iOS versions. Capacitor 2.0 supports iOS 11+. Capacitor uses the WKWebView. 

Android version support for Capacitor is more complex than iOS. Currently, we are targeting API level 21 or greater, meaning Android 5.0 (Lollipop) or above. This represents over 90% of the Android market.

Also, Capacitor requires an Android WebView with Chrome version 50 or later. On Android 5 and 6, the Capacitor uses the System WebView. On Android 7+, Google Chrome is used.

**Reply by distante** [reply]
With the current problems that some devices have with the webview and accessibility it would be a good fix to deploy a crosswalk alternative for capacitor.
